chainer/variable.py

```diff
@@ -1578,6 +1600,23 @@ def add_cand(cand):
     grads.assert_no_grads()
 
 
+def _unsafe_variable(data, name=None, grad=None, requires_grad=True,
```

Comment:
How about making this a `@staticmethod` of `Variable`?
The name could be arguable. How about `Variable._init_unchecked`? (`init` in order to express it's an alternative to `__init__`. `unchecked` because it's more clearly express the behavior than `unsafe`).